### PR TITLE
Feat: Map and Footer Added

### PR DESCRIPTION
**Pull request by Rushi67**

### CODE REVIEW

**Comment by psantosh16** [comment]
Make suggested changes , drop a message

**Review by psantosh16 — CHANGES_REQUESTED** [review]
Make suggested changes , drop a message

**Comment by Rushi67** [comment]
Done

File: src/components/footer/footer.jsx
Comment (psantosh16) [inline src/components/footer/footer.jsx:1]: Extract the Icons List from main footer file and make another file for that in root footer folder.

Maintain clean code

**Review by psantosh16 — CHANGES_REQUESTED** [review]

File: src/components/footer/footer.jsx
Comment (Rushi67) [reply src/components/footer/footer.jsx:1]: done

**Review by psantosh16 — APPROVED** [review]

**Pull request merged**